src/main/java/frc/robot/lib/interfaces/Swerve.java

```diff
@@ -122,6 +124,14 @@ public static void periodic() {
                 rotation
             ));
         }
+
+        for (int i = 0; i < 2; i++) {
+            loadingStationPoses.add(new Pose2d(
+                starting_loading_station.getX(),
+                Constants.TargetPoses.BLUE_SUBSTATION_Y + i * Constants.TargetPoses.SUBSTATION_SPACING,
```

Comment:
So this will define the Y coordinates of the substation poses starting at the lowest Y position. This is fine, but we need to change the BLUE_SUBSTATION_Y constant to be the lower Y position. At the moment I think it is the higher Y position.